### DISCUSSION TITLE
Run Synpress with CircleCI and TestRail

### DISCUSSION THREAD

**Discussion by daviti** [Synthetixio/synpress#401] [Q&A] [answered]
I hope someone has tried to do this and hope some one has an answer. I am trying to automate with CI/CD and want to set up CircleCi with Synpress/Cypress but I am not able to. Everything I try fails. Has anybody been able to do this?? Also, has anybody been able to set up TestRail with Synpress??

**Comment by drptbl** [comment, marked as answer]
> I hope someone has tried to do this and hope some one has an answer. I am trying to automate with CI/CD and want to set up CircleCi with Synpress/Cypress but I am not able to. Everything I try fails. Has anybody been able to do this?? Also, has anybody been able to set up TestRail with Synpress??

I have an example of how to set it up on Github Actions here:
https://github.com/Synthetixio/staking/blob/dev/.github/workflows/audit_build_verify.yml#L104-L179

It works in CI for sure if you set it up correctly.

**Reply by daviti** [reply]
Thank you this is a great help. I will try to set it up. One more question, I supposed that the setting for CI is similar to the name: Staking CI file?
